global-news-app/src/components/Search/RecentSearchesDropdown/RecentSearchesDropdown.style.tsx

```diff
@@ -0,0 +1,56 @@
+import RemoveHistoryIcon from "assets/icons/RemoveHistoryIcon";
+import styled from "styled-components";
+
+export const DropdownWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  background: ${({ theme }) => theme.colors.white};
+  border-radius: 10px;
+
+  position: absolute;
+  top: 55px;
+  width: 100%;
+  left: 0;
+  padding: 1rem;
+`;
+
+export const DropdownHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  p {
+    font-weight: 500;
+    font-size: 12px;
```

Comment:
for spacings that are repetitive consider to add a const, or add to the global theme